wire-ios/Wire-iOS/Sources/UserInterface/ScreenCurtain/ScreenCurtainView.swift

```diff
@@ -0,0 +1,40 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+
+struct ScreenCurtainView: View {
+
+    var body: some View {
+        VStack {
+            Spacer()
+            HStack {
+                Spacer()
+                // for some reason `Image(.Wire.shield)` didn't show the shield
```

Comment:
on preview or device?

Reply:
On the device. It's weird, I spent a while finding the cause why the image wasn't visible.

Reply:
```
Image(.Wire.shield)
```

<img width="250" alt="Screenshot 2024-05-06 at 9 48 59 AM" src="https://github.com/wireapp/wire-ios/assets/15628584/3aa5079e-119f-45cf-bcba-626cb4012742">

```
Image(uiImage: .init(resource: .Wire.shield))
```

<img width="250" alt="Screenshot 2024-05-06 at 9 49 29 AM" src="https://github.com/wireapp/wire-ios/assets/15628584/8663946d-cf6a-492f-9a4b-14fbc213ee32">